osfs/os_js.go

```diff
@@ -0,0 +1,23 @@
+package osfs
```

Comment:
Since in go-git, you can open and clone a repository declaring the kind of fs, I rather having this implementation as an implementation that returns "Not Implemented" errors for every method or one that simply panics if you call to os.New()

Reply:
This fixes compile errors.